lib/scope.js

```diff
@@ -826,36 +838,73 @@ module.exports = {
     // data for `users[0]`, there's no need to check the trigger var.
     // That way devs with simple cases can ignore the trigger var column.
 
-    // If field uses proxies and there has been more than one match
-    // we now use the trigger var to narrow down the matches.
-    if ( uses_proxies && matches.length > 1 ) {
```

Comment:
This was the line that was causing the problem, but fixing it meant I had to change other things so they wouldn't break.